.github/workflows/deploy.yml

```diff
@@ -37,9 +38,8 @@ jobs:
       
       - name: Check Release Tag Format
         run: |
-          re='^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-(beta|alpha|rc))(\.[0-9]+)?$'
-          if ! [[ $RELEASE_VERSION =~ $re ]]; then
-            echo "Tag does not match expected regex pattern for beta releases ($re)"
+          if ! [[ $RELEASE_VERSION =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-(beta|alpha|rc))(\.[0-9][0-9]*)?$ ]]; then
+            echo 'Tag does not match expected regex pattern for beta releases (^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(-(beta|alpha|rc))(\.[0-9][0-9]*)?$)'
```

Comment:
what was wrong with the `$re`?

Reply:
Bash is what was wrong

Reply:
bash would try to run the stuff within the parenthesis as a subshell command. This was just easier.